src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,21 @@
+.cards {
+  --columns: 1;
+  max-width: 944px;
+  display: grid;
+  row-gap: 46px;
+  column-gap: 48px;
+  grid-template-columns: repeat(var(--columns), 1fr);
+  margin: 0 auto;
+
+  @media (min-width: 488px) {
+    --columns: 2;
+  }
+
+  @media (min-width: 768px) {
+    --columns: 3;
+  }
+
+  @media (min-width: 1024px) {
+    --columns: 4;
+  }
```

Comment:
Now everything is written correctly, but it is better not to change the value of the variable depending on the size, but to change the class parameter itself.